concurrency-primer.tex

```diff
@@ -321,6 +321,9 @@ \section{Enforcing law and order}
 This model, defined by Leslie Lamport in 1979,
 is called \introduce{sequential consistency}.
 
+Notice that using atomic variables as an lvalue expression, such as \monobox{v\_ready = true} and \monobox{while(!v\_ready)}, is a convenient alternative to explicitly using \monobox{atomic\_load} or \monobox{atomic\_store}. 
```

Comment:
Code generation on atomic load and store is a little bit tricky. As explained in [LLVM's document](https://llvm.org/docs/Atomics.html#id16), sometimes they are just normal load/store, without atomic instructions, and their behavior can vary across different architectures. Should we explain more on this? Or readers may be confused on examples in section 7 and 10.1

Reply:
Can you summarize the above in the footnote?

Reply:
Resolved in 4dc5699.